app/views/middleware_server/_add_datasource_step1.html.haml

```diff
@@ -32,19 +31,18 @@
                                             "ng-required"  => "true",
                                             "ng-model"     => "step1DsModel.jndiName"}
 
-      .col-sm-2
-      .col-sm-10
+      .col-sm-8.col-sm-offset-4
         %p.ng-error-messages{"ng-show" => "step1Form.jndi_name_input.$error.required"}
           = _("Required")
         %p.ng-error-messages{"ng-show" => "step1Form.jndi_name_input.$error.maxlength"}
           = _("Maximum length is 256")
 
     .modal-footer
-      %button.btn.btn-default.dialog-cancel-button{:type          => "button",
-                                                   "ng-click"     =>"reset()"}
+      %button.btn.btn-default.dialog-cancel-button{:type      => "button",
+                                                   "ng-click" =>"reset()"}
```

Comment:
@mtho11 please add a space after hash rocket, i noticed that in couple of other places.